core/src/main/java/org/apache/iceberg/rest/auth/RefreshingAuthManager.java

```diff
@@ -18,73 +18,27 @@
  */
 package org.apache.iceberg.rest.auth;
 
-import java.util.List;
-import java.util.concurrent.Future;
 import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
 import javax.annotation.Nullable;
 import org.apache.iceberg.util.ThreadPools;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
- * An {@link AuthManager} that provides machinery for refreshing authentication data asynchronously,
- * using a background thread pool.
+ * @deprecated since 1.9.0, will be removed in 1.10.0; use {@link ThreadPools#getAuthRefreshPool()}.
```

Comment:
version info needs to be updated here, since the deprecation applies with 1.10.0 and we can only remove with 1.11.0